apps/pixeloven/site/src/server/views/Body/Body.tsx

```diff
@@ -0,0 +1,41 @@
+import { Script } from "@server/views";
+import { State } from "@shared/store/types";
```

Comment:
This is failing because it doesn't exist

```
ERROR in /Users/jimmyw/repos/jimmynotjim/pixeloven/apps/pixeloven/site/src/server/views/Body/Body.tsx
[tsl] ERROR in /Users/jimmyw/repos/jimmynotjim/pixeloven/apps/pixeloven/site/src/server/views/Body/Body.tsx(2,23)
      TS2307: Cannot find module '@shared/store/types'.
```

Reply:
Fixed